fcrepo-exts/migration-utils
# Point at recent fcrepo-storage-ocfl snapshot with ocfl-java 2.0.1 changes

**JIRA Ticket**: https://fedora-repository.atlassian.net/browse/FCREPO-3936

# What does this Pull Request do?
Updates pom file to point to current fcrepo-storage-ocfl snapshot, and updates package names to correspond to changes in https://github.com/fcrepo/fcrepo-storage-ocfl/pull/56

# How should this be tested?

This should have no functional changes, it should still build and work. You may need to pull down https://github.com/fcrepo/fcrepo-storage-ocfl/pull/56 and build that locally first.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @fcrepo-exts/committers
